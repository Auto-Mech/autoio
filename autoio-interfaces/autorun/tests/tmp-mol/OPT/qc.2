! 
memory,128,m
nosym
angstrom
geometry = {
N    0.255878  -0.289694   0.000000
N    1.290839   0.166775   0.000000
O   -1.367199   0.101038   0.000000
}
set,spin=2
set,charge=0
basis=aug-cc-pvdz
nn(1)=1
nn(2)=2
nn(3)=3
{rhf,,maxit=300}
{casscf,}
{rs2,}

status,all,crash

molpro_energy=energy
show[1,e25.15],molpro_energy
{force,varsav}
text,MOLGRAD
table,nn,gradx,grady,gradz
ftyp,f,d,d,d
