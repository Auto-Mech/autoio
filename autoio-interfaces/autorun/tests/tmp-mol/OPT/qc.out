
 Primary working directories    : /scratch/kmoore
 Secondary working directories  : /scratch/kmoore
 Wavefunction directory         : /home/kmoore/wfu/
 Main file repository           : /scratch/kmoore/

 SHA1      : f86d681fdec979508940bfbbc93fd7db66630888
 NAME      : 2015.1.24
 ARCHNAME  : linux/x86_64
 FC        : /blues/gpfs/home/software/spack-0.10.1/opt/spack/linux-centos7-x86_64/intel-17.0.4/intel-mpi-2017.3-dfphq6kavje2olnichisvjjndtridrok/compilers_and_libraries_2017.4.196/linux/mpi/intel64/bin/mpiifort
 BLASLIB   : -L/blues/gpfs/home/software/spack-0.10.1/opt/spack/linux-centos7-x86_64/intel-17.0.4/intel-mkl-2017.3.196-jyjmyutcol3veyt4bazftyqkxxzmfd7o/pkg/compilers_and_libraries_2017.4.196/linux/mkl/lib/intel64 -lmkl_intel_ilp64 -lmkl_intel_thread -lmkl_core
 id        : anlgov

 Nodes     nprocs
 b460         3
 Number of processes for MPI-2 version of Molpro:   nprocs(total)=    4   nprocs(compute)=    3   nprocs(helper)=    1

 Using customized tuning parameters: mindgm=7; mindgv=16; mindgc=1; mindgr=1; noblas=0; minvec=7
 default implementation of scratch files=sf  

                                                                                 !
 memory,128,m
 nosym
 angstrom
 geometry = {
 N    0.255878  -0.289694   0.000000
 N    1.290839   0.166775   0.000000
 O   -1.367199   0.101038   0.000000
 }
 set,spin=0
 set,charge=0
 basis=aug-cc-pvdz
 nn(1)=1
 nn(2)=2
 nn(3)=3
 {rhf,,maxit=300}
 {casscf,}
 {rs2,}
 
 status,all,crash
 
 molpro_energy=energy
 show[1,e25.15],molpro_energy
 {force,varsav}
 text,MOLGRAD
 table,nn,gradx,grady,gradz
 ftyp,f,d,d,d

 Variables initialized (889), CPU time= 0.01 sec
 Commands  initialized (702), CPU time= 0.02 sec, 572 directives.
 Default parameters read. Elapsed time= 0.19 sec

 Checking input...
 Passed
1


                                         ***  PROGRAM SYSTEM MOLPRO  ***
                                       Copyright, TTI GmbH Stuttgart, 2015
                                    Version 2015.1 linked Sep 21 2017 09:51:13


 **********************************************************************************************************************************
 LABEL *                                                                                 
  64 bit mpp version                                                                     DATE: 16-Aug-21          TIME: 21:01:21  
 **********************************************************************************************************************************

 SHA1:             f86d681fdec979508940bfbbc93fd7db66630888
 **********************************************************************************************************************************

 Variable memory set to  128000000 words,  buffer space   230000 words

 ZSYMEL=NOSYM
 ZUNIT=ANG
 

Geometry recognized as XYZ

 SETTING SPIN           =     0.00000000D+00                                  
 SETTING CHARGE         =     0.00000000D+00                                  
 SETTING BASIS          =    AUG-CC-PVDZ
 SETTING NN(1)          =         1.00000000                                  
 SETTING NN(2)          =         2.00000000                                  
 SETTING NN(3)          =         3.00000000                                  


 Recomputing integrals since basis changed


 Using spherical harmonics

 Library entry N      S aug-cc-pVDZ          selected for orbital group  1
 Library entry N      P aug-cc-pVDZ          selected for orbital group  1
 Library entry N      D aug-cc-pVDZ          selected for orbital group  1
 Library entry O      S aug-cc-pVDZ          selected for orbital group  2
 Library entry O      P aug-cc-pVDZ          selected for orbital group  2
 Library entry O      D aug-cc-pVDZ          selected for orbital group  2


 PROGRAM * SEWARD (Integral evaluation for generally contracted gaussian basis sets)     Author: Roland Lindh, 1990

 Geometry written to block  1 of record 700


 Point group  C1  



 ATOMIC COORDINATES

 NR  ATOM    CHARGE       X              Y              Z

   1  N       7.00    0.483539341   -0.547442320    0.000000000
   2  N       7.00    2.439332181    0.315159074    0.000000000
   3  O       8.00   -2.583631668    0.190934148    0.000000000

 Bond lengths in Bohr (Angstrom)

 1-2  2.137570302  1-3  3.154796001
     ( 1.131153491)     ( 1.669446149)

 Bond angles

  2-1-3  142.66458375

 NUCLEAR CHARGE:                   22
 NUMBER OF PRIMITIVE AOS:         111
 NUMBER OF SYMMETRY AOS:          105
 NUMBER OF CONTRACTIONS:           69   (  69A   )
 NUMBER OF CORE ORBITALS:           3   (   3A   )
 NUMBER OF VALENCE ORBITALS:       12   (  12A   )


 NUCLEAR REPULSION ENERGY   51.81936239

 EXTRA SYMMETRY OF AOS IN SYMMETRY 1:   1 1 1 1 1 1 2 1 1 2   1 1 2 1 1 2 1 2 1 1   2 1 2 1 1 1 1 1 1 2   1 1 2 1 1 2 1 1 2 1
                                        2 1 1 2 1 2 1 1 1 1   1 1 2 1 1 2 1 1 2 1   1 2 1 2 1 1 2 1 2

 Eigenvalues of metric

         1 0.133E-03 0.570E-03 0.266E-02 0.465E-02 0.104E-01 0.106E-01 0.184E-01 0.325E-01


 Contracted 2-electron integrals neglected if value below      1.0D-11
 AO integral compression algorithm  1   Integral accuracy      1.0D-11

     9.961 MB (compressed) written to integral file ( 31.1%)

     Node minimum: 3.146 MB, node maximum: 3.670 MB


 NUMBER OF SORTED TWO-ELECTRON INTEGRALS:     973245.     BUFFER LENGTH:  32768
 NUMBER OF SEGMENTS:   1  SEGMENT LENGTH:     973245      RECORD LENGTH: 524288

 Memory used in sort:       1.53 MW

 SORT1 READ     4039007. AND WROTE      481742. INTEGRALS IN      2 RECORDS. CPU TIME:     0.04 SEC, REAL TIME:     0.04 SEC
 SORT2 READ     1453813. AND WROTE     2917320. INTEGRALS IN     42 RECORDS. CPU TIME:     0.03 SEC, REAL TIME:     0.04 SEC

 Node minimum:      971635.  Node maximum:      973245. integrals

 OPERATOR DM      FOR CENTER  0  COORDINATES:    0.000000    0.000000    0.000000


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      18       19.29       500      610      700      900      950      970     1000      129      960     1100   
                                          VAR    BASINP    GEOM    SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1700   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    OPER   

 PROGRAMS   *        TOTAL       INT
 CPU TIMES  *         0.36      0.18
 REAL TIME  *         0.91 SEC
 DISK USED  *        32.26 MB      
 GA USED    *         0.00 MB       (max)       0.00 MB       (current)
 **********************************************************************************************************************************


 PROGRAM * RHF-SCF (CLOSED SHELL)       Authors: W. Meyer, H.-J. Werner


 NUMBER OF ELECTRONS:      11+   11-    SPACE SYMMETRY=1    SPIN SYMMETRY: Singlet 
 CONVERGENCE THRESHOLDS:    1.00E-06 (Density)    1.00E-07 (Energy)
 MAX. NUMBER OF ITERATIONS:      300
 INTERPOLATION TYPE:            DIIS
 INTERPOLATION STEPS:              2 (START)      1 (STEP)
 LEVEL SHIFTS:                  0.00 (CLOSED)  0.00 (OPEN) 



 Orbital guess generated from atomic densities. Full valence occupancy:   15

 Molecular orbital dump at record        2100.2

 Initial occupancy:  11

 ITERATION    DDIFF          GRAD             ENERGY        2-EL.EN.            DIPOLE MOMENTS         DIIS   ORB.
    1      0.000D+00      0.000D+00      -183.39629974    232.292571    2.40803   -0.55758   -0.00000    0    start
    2      0.000D+00      0.162D-01      -183.46571069    235.494814    0.01042    0.30812   -0.00000    1    diag
    3      0.336D-01      0.162D-01      -183.61502559    235.497166    0.87671   -0.08688   -0.00000    2    diag
    4      0.173D-01      0.234D-02      -183.62659550    235.223038    0.77775   -0.06530    0.00000    3    diag
    5      0.474D-02      0.110D-02      -183.62952233    235.440084    0.67002   -0.08041    0.00000    4    diag
    6      0.294D-02      0.392D-03      -183.62977718    235.481293    0.66899   -0.06544    0.00000    5    diag
    7      0.934D-03      0.253D-03      -183.62983314    235.453161    0.64490   -0.07168    0.00000    6    diag
    8      0.362D-03      0.661D-04      -183.62984206    235.458353    0.64535   -0.07232   -0.00000    7    diag
    9      0.151D-03      0.277D-04      -183.62984534    235.459080    0.64538   -0.07259   -0.00000    8    diag
   10      0.921D-04      0.177D-04      -183.62984747    235.458895    0.64589   -0.07302   -0.00000    9    orth
   11      0.832D-04      0.120D-04      -183.62984908    235.459035    0.64648   -0.07340    0.00000    9    diag
   12      0.106D-03      0.598D-05      -183.62984946    235.459031    0.64693   -0.07366    0.00000    9    diag
   13      0.641D-04      0.231D-05      -183.62984950    235.458992    0.64702   -0.07370    0.00000    9    diag
   14      0.164D-04      0.112D-05      -183.62984950    235.458990    0.64707   -0.07371    0.00000    9    diag
   15      0.703D-05      0.368D-06      -183.62984951    235.458971    0.64707   -0.07371    0.00000    9    diag
   16      0.227D-05      0.947D-07      -183.62984951    235.458972    0.64707   -0.07371    0.00000    9    diag
   17      0.394D-06      0.396D-07      -183.62984951    235.458974    0.64708   -0.07371   -0.00000    0    orth

 Final occupancy:  11

 !RHF STATE 1.1 Energy               -183.629849505333
 Nuclear energy                        51.81936239
 One-electron energy                 -353.17869909
 Two-electron energy                  117.72948720
 Virial quotient                       -1.00334427
 !RHF STATE 1.1 Dipole moment           0.64707672    -0.07370894    -0.00000000
 Dipole moment /Debye                   1.64459725    -0.18733716    -0.00000000

 Orbital energies:

         1.1          2.1          3.1          4.1          5.1          6.1          7.1          8.1          9.1         10.1
    -20.665108   -15.774493   -15.758037    -1.507559    -1.268087    -0.831010    -0.681319    -0.661886    -0.661073    -0.526252

        11.1         12.1         13.1
     -0.525038     0.014761     0.093692

 HOMO     11.1    -0.525038 =     -14.2870eV
 LUMO     12.1     0.014761 =       0.4017eV
 LUMO-HOMO         0.539800 =      14.6887eV


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      18       19.29       500      610      700      900      950      970     1000      129      960     1100   
                                          VAR    BASINP    GEOM    SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1700   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    OPER   

              2       4        2.88       700     1000      520     2100   
                                         GEOM     BASIS   MCVARS     RHF  

 PROGRAMS   *        TOTAL       RHF       INT
 CPU TIMES  *         0.58      0.22      0.18
 REAL TIME  *         1.23 SEC
 DISK USED  *        99.95 MB      
 SF USED    *         0.02 MB      
 GA USED    *         0.00 MB       (max)       0.00 MB       (current)
 **********************************************************************************************************************************


 PROGRAM * MULTI (Direct Multiconfiguration SCF)       Authors: P.J. Knowles, H.-J. Werner (1984)     S.T. Elbert (1988)


 Number of closed-shell orbitals:   3 (   3 )
 Number of active  orbitals:       12 (  12 )
 Number of external orbitals:      54 (  54 )

 State symmetry 1

 Number of electrons:    16    Spin symmetry=Singlet   Space symmetry=1
 Number of states:        1
 Number of CSFs:      70785   (245025 determinants, 245025 intermediate states)

 Molecular orbitals read from record     2100.2  Type=RHF/CANONICAL (state 1.1)

 EXTRA SYMMETRY OF MOS IN SYMMETRY 1:   1 1 1 1 1 1 1 1 2 2   1 1 2 1 1 1 2 1 1 2   1 2 1 1 1 1 2 1 1 1   2 1 2 1 1 1 2 1 2 1
                                        2 1 1 2 1 1 2 1 1 2   1 1 2 2 1 1 1 2 1 1   1 2 1 1 2 1 2 1 1

 Wavefunction dump at record             2140.2

 Convergence thresholds  0.10E-01 (gradient)  0.10E-05 (energy)  0.10E-02 (step length)

 Number of orbital rotations:      846   (    36 Core/Active    162 Core/Virtual   0 Active/Active    648 Active/Virtual)
 Total number of variables:     245871


 ITER. MIC  NCI  NEG     ENERGY(VAR)     ENERGY(PROJ)   ENERGY CHANGE     GRAD(0)  GRAD(ORB)   GRAD(CI)     STEP       TIME

   1   50   26    0    -183.67637684    -183.83336887   -0.15699203    0.06191312 0.00011774 0.00994133  0.14D+01      7.29
   2   26   39    0    -183.82936211    -183.83100673   -0.00164462    0.03273791 0.00000687 0.00036824  0.13D+00     19.50
   3   27   18    0    -183.83100826    -183.83100827   -0.00000001    0.00013554 0.00000243 0.00003001  0.26D-03     24.43

 ** WVFN ****  CONVERGENCE REACHED, FINAL GRADIENT:  0.90D-05


 First order charge density matrix for state 1.1 saved on record  2140.2 (density set 1)

 Results for state 1.1
 =====================

 !MCSCF STATE 1.1 Energy             -183.831008272784
 Nuclear energy                        51.81936239
 Kinetic energy                       183.41780053
 One electron energy                 -353.39989115
 Two electron energy                  117.74952049
 Virial ratio                           2.00225282

 !MCSCF STATE 1.1 Dipole moment         0.71760501    -0.11212266    -0.00000000
 Dipole moment /Debye                   1.82385055    -0.28496872    -0.00000000


 EXTRA SYMMETRY OF MOS IN SYMMETRY 1:   1 1 1 1 1 1 2 1 1 2   1 1 2 1 1 1 1 1 2 1   2 1 2 1 1 1 1 2 1 1   2 1 2 1 1 2 1 1 2 1
                                        2 1 1 1 2 1 2 1 1 2   1 1 2 2 1 1 1 2 1 1   1 2 1 1 2 1 2 1 1

 Natural orbital dump at molpro section  2140.2    (Orbital set 2)



 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      19       26.25       500      610      700      900      950      970     1000      129      960     1100   
                                          VAR    BASINP    GEOM    SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1700     1380   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    OPER     JKOP   

              2       5        5.58       700     1000      520     2100     2140   
                                         GEOM     BASIS   MCVARS     RHF     MCSCF   

 PROGRAMS   *        TOTAL    CASSCF       RHF       INT
 CPU TIMES  *        28.59     28.01      0.22      0.18
 REAL TIME  *        33.12 SEC
 DISK USED  *        99.95 MB      
 SF USED    *       161.21 MB      
 GA USED    *         0.00 MB       (max)       0.00 MB       (current)
 **********************************************************************************************************************************


 PROGRAM * RS2 (Multireference RS Perturbation Theory)     Authors: H.-J. Werner (1993), P. Celani (1998)


 Convergence thresholds:  THRVAR = 1.00D-10  THRDEN = 1.00D-08

 Number of optimized states:  1  Roots:   1
 Number of reference states:  1  Roots:   1

 Reference symmetry:                   1   Singlet 
 Number of electrons:                 22
